internal/services/springcloud/spring_cloud_dynatrace_application_performance_monitoring_resource_test.go

```diff
@@ -161,10 +161,10 @@ resource "azurerm_spring_cloud_dynatrace_application_performance_monitoring" "te
   spring_cloud_service_id = azurerm_spring_cloud_service.test.id
   globally_enabled        = true
   api_url                 = "https://test-api-url.com"
-  api_token               = "test-api-token"
+  api_token               = "dt0s01.ST2EY72KQINMH574WMNVI7YN.G3DFPBEJYMODIDAEX454M7YWBUVEFOWKPRVMWFASS64NFH52PX6BNDVFFM572RZM"
```

Comment:
this might be better as 
```
  api_token               = "dt0s01.AAAAAAAAAAAAAAAAAAAAAAAA.BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
```